pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -239,6 +241,10 @@ func (cr *PerconaXtraDBCluster) Validate() error {
 			if len(channel.SourcesList) == 0 {
 				return errors.Errorf("sources list for replication channel %s should be empty, because it's replica", channel.Name)
 			}
+
+			if channel.Config.SSL && channel.Config.CA == "" {
```

Comment:
we need a `nil` check for `channel.Config` here

Reply:
done. sorry for the delay